deploy/ansible/poc_bom_iteration_playbook.yml

```
@@ -5,10 +5,30 @@
     sapbits_location_base_path: "https://npeus2saplibb546.blob.core.windows.net/sapbits"
     bom_base_name: "S4HANA_1909_SP2_v002"
     target_media_location: /usr/sap/install
-    app_sid: Z01
+    aas_hostname:
+    aas_instance_number:
+    app_sid: "Z01"
```

Very minor request, default demo SID in beta branch deployments is `X00`, could we change to that?

```suggestion
    app_sid: "X00"
```